docs/monetization/App_Subscriptions.md

```diff
@@ -36,6 +36,33 @@ When a user subscribes to your app, there are a few things you will need to impl
 
 ### Gating Premium Interactions
 
+Each interaction payload includes an `entitlements` field containing an array of full [entitlement objects](#DOCS_MONETIZATION_ENTITLEMENTS/entitlement-object). You can use this field to determine if the user or guild is subscribed to your app. If the user or guild is not subscribed and you wish to present them with a means to do so you can use a [button](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/buttons) with a [premium style](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/button-object-button-styles) and a `sku_id` attached. You may also use these buttons to present users with options to make a [One-Time Purchase](#DOCS_MONETIZATION_ONE-TIME_PURCHASES).
```

Comment:
This gets restated below, under `Type 10 Interaction Response`: 
https://github.com/discord/discord-api-docs/blob/8588250bf8543b25638bac0cf0f5268f9205b138/docs/monetization/App_Subscriptions.md?plain=1#L77-L81

Reply:
yeah it's relevant in both cases.

Reply:
> yeah it's relevant in both cases.

it just seems redundant, since it's already started once here and then again under the sub header on the same page